include/boost/exception/detail/error_info_impl.hpp

```diff
@@ -46,6 +47,13 @@ boost
         typedef T value_type;
 
         error_info( value_type const & value );
+#ifndef BOOST_NO_CXX11_RVALUE_REFERENCES
+        error_info( value_type && value);
+        error_info( error_info const & ) = default;
```

Comment:
Also, Visual C++ 12 doesn't support defaulted move constructors and move assignment operators even though it supports defaulted functions in general.